backend/app/adapter/sqldb/migration/29_create_incident_table.sql

```diff
@@ -0,0 +1,10 @@
+-- +migrate Up
+CREATE TABLE incident
+(
+    id          CHARACTER VARYING(4)      NOT NULL UNIQUE,
```

Comment:
```suggestion
    id          VARCHAR(4) NOT NULL PRIMARY KEY,
```
id is the only way to uniquely `INDENTIFY` a incident. So we should it as the `PRIMARY KEY`.